scripts/ci-footprint-and-performance/tests/010-hwlatdetect.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck source=scripts/ci-footprint-and-performance/tests/functions.sh
+source "${SCRIPTDIR}/functions.sh"
+
+TEST_DURATION=$(( 2 * 60 ))
```

Comment:
Isn't 2m too short? I would run at least 5m each, possibly even more time.

Reply:
Don't think so, default value is 10s and in RHEL docs I've seen 60 seconds. This is just sanity for additional results if we see something weird in oslat. Oslat is the key test here.